ui/00_message/common/job_info.xml

```diff
@@ -8,59 +8,89 @@
     <original>攻守のバランスに長けた戦場の花形
 盾を駆使したカウンター戦法を得意とする
 常に前線に斬り込んでいける安定感が持ち味</original>
+    <translation>Balances offense and defense in battle.
+Can use the shield to counter attacks.
+The Leader of the Battlefield.</translation>
   </message>
   <message>
     <key>JOB_INFO_2</key>
     <original>ヒット＆アウェイのかく乱戦を得意とする軽業師
 ロープワークで死角に回り込み、瞬時に巨大敵の
 頭上に迫り連撃を加える様は戦場の疾風</original>
+    <translation>An Acrobat who excels at hit and run tactics.
+Uses a rope to instantly move to the blind spot of large enemies.
+The Gale of the Battlefield.</translation>
   </message>
   <message>
     <key>JOB_INFO_3</key>
     <original>精密な射撃で遠距離から先制するスナイパー役
 特殊矢による多様な支援力も魅力
 強敵の弱点を正確に射ぬき戦況を優位に導く</original>
+    <translation>Able to deal precise damage from a long distance.
+Has a variey of support abilities by the use of Special arrows.
+The Sniper of the Battlefield.</translation>
   </message>
   <message>
     <key>JOB_INFO_4</key>
     <original>回復と支援魔法で仲間を支える癒しの術師
 その術は味方へのさまざまな恩恵だけでなく
 怒れる敵の“隠された弱点”すら看破する</original>
+    <translation>Supports the party with healing and support magick.
+Can not only heal, but also reveal weakness of enemies.
+The Pillar of the Battlefield.</translation>
   </message>
   <message>
     <key>JOB_INFO_5</key>
     <original>敵の注目を一手に集め反撃する強固な砦役
 大盾で受けた敵の力を攻撃・支援の術に変換
 ５属性を操り、仲間を属性面でもフォローする</original>
+    <translation>Attract attention and protect others with your strong defense.
+Convert your enemies power into magick to attack and support.
```

Comment:
Yes it is intentional, I thought that for a small description like this it might be a bit complicated to explain that you can use the 5 elements but you also have non elemental support and attack magic.

I could just change it to "...power into elemental magick..."